### PR TITLE
Initial post comment

### PR DESCRIPTION
**Pull request by shiara02**
### OVERVIEW

-User can add comments on posts

### NOTES

-Pagination is not yet included
-Php artisan migrate

### SNIPPETS

image
image
image

### CODE REVIEW

File: src/app/Http/Controllers/CommentController.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+use App\Models\Post;
+use App\Models\Comment;
+use Illuminate\View\View;
+use App\Http\Requests\CommentRequest;
+use Illuminate\Http\RedirectResponse;
+
+class CommentController extends Controller
+{
+    public function index(Post $post): View
```
Comment (hipe01mark) [inline src/app/Http/Controllers/CommentController.php:26]: add comment

File: src/app/Http/Controllers/CommentController.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    {
+        return view('comment.index', ['post' => $post]);
+    }
+
+    public function store(CommentRequest $request, Post $post): RedirectResponse
```
Comment (hipe01mark) [inline src/app/Http/Controllers/CommentController.php:35]: add comment

File: src/app/Http/Controllers/CommentController.php (hunk 1 above)
Comment (hipe01mark) [reply src/app/Http/Controllers/CommentController.php:26]: create service layer

File: src/app/Http/Controllers/CommentController.php (hunk 2 above)
Comment (hipe01mark) [reply src/app/Http/Controllers/CommentController.php:35]: create service layer

File: src/app/Http/Controllers/CommentController.php (hunk 1 above)
Comment (shiara02) [reply src/app/Http/Controllers/CommentController.php:26]: I have updated my PR, sir. Thank you! 🙇

File: src/app/Http/Controllers/CommentController.php (hunk 2 above)
Comment (shiara02) [reply src/app/Http/Controllers/CommentController.php:35]: I have updated my PR, sir. Thank you! 🙇

**Review by hipe01mark — DISMISSED** [review]
LGTM 🙇

File: src/app/Services/CommentService.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+namespace App\Services;
+
+use App\Models\Post;
+use App\Models\Comment;
+use App\Http\Requests\CommentRequest;
+use Illuminate\Database\Eloquent\Collection;
+
+class CommentService
+{
+    public function getCommentsForPost(Post $post): Collection
```
Comment (hipe01mark) [inline src/app/Services/CommentService.php:15]: Add comment!

File: src/app/Services/CommentService.php [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    {
+        return $post->comments;
+    }
+
+    public function storeComment(CommentRequest $request, Post $post): Comment
```
Comment (hipe01mark) [inline src/app/Services/CommentService.php:23]: Add comment!

File: src/app/Services/CommentService.php (hunk 3 above)
Comment (shiara02) [reply src/app/Services/CommentService.php:15]: I have resolved the comment, sir! Thank you. 🙇

File: src/app/Services/CommentService.php (hunk 4 above)
Comment (shiara02) [reply src/app/Services/CommentService.php:23]: I have resolved the comment, sir! Thank you. 🙇

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Review by bpocrafael — APPROVED** [review]

**Pull request merged**